integration-tests/smoke/keeper_test.go

```diff
@@ -98,7 +98,7 @@ func getKeeperSuite(
 					environment.NewChainlinkConfig(
 						environment.ChainlinkReplicas(6, config.ChainlinkVals()),
 						"chainlink-keeper-core-ci",
-						config.GethNetworks()...,
+						environment.PerformanceGeth,
```

Comment:
Please use environment.PerformanceGeth for Keepers and VRF types of tests, otherwise it can OOM

Reply:
Will do from now on, thanks for the help!